server/data/db_schema.sql

```diff
@@ -17,24 +19,30 @@
 -- Table structure for table `apikey`
 --
 
+DROP TABLE IF EXISTS `apikey`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
-CREATE TABLE IF NOT EXISTS `apikey` (
+CREATE TABLE `apikey` (
   `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
   `type` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   `name` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   `token` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `can_create_users` tinyint(1) unsigned DEFAULT NULL,
+  `can_create_tickets` tinyint(1) unsigned DEFAULT NULL,
+  `can_check_tickets` tinyint(1) unsigned DEFAULT NULL,
+  `should_return_ticket_number` tinyint(1) unsigned DEFAULT NULL,
   PRIMARY KEY (`id`)
-) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+) ENGINE=InnoDB AUTO_INCREMENT=12 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
`12` is suspicious here 🤔 what does it mean to have an auto increment of 12 instead of 1? Does it go: 1, 13, 25, 37, etc.?